src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -89,6 +95,10 @@ const LiveRadioPage = ({ pageData }) => {
     translations,
   );
 
+  const { enabled } = useToggle('liveRadioSchedule');
+  const showSchedule =
+    enabled && radioScheduleOnPage && masterBrand !== 'bbc_persian_radio';
```

Comment:
Working on a way to not hardcode Persian here

Reply:
Leaving this until toggles are on iSite next week - https://github.com/orgs/bbc/projects/53#card-42064710